### PR TITLE
feature/DAT-73030

### PR DESCRIPTION
**Pull request by theblushingcrow**
## Description

Segmented Button is now typed. It can also accept an input of border radius, overriding the default (4px). 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [x] no

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datorama/app-components",
-  "version": "1.5.0",
+  "version": "1.6.0",
```
Comment (crazypixel) [inline package.json]: can you bump to 1.7.0

File: src/components/SegmentedButton/SegmentedButton.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { hexToRgba, Tooltip } from '../..';
```
Comment (crazypixel) [inline src/components/SegmentedButton/SegmentedButton.tsx:3]: can you check that in build we don't have a circular dep. cause this is imported in index.js and importing from index.js

**Review by crazypixel — APPROVED** [review]
looks good. added 2 comments.

File: package.json (hunk 1 above)
Comment (theblushingcrow) [reply package.json]: done

File: src/components/SegmentedButton/SegmentedButton.tsx (hunk 2 above)
Comment (theblushingcrow) [reply src/components/SegmentedButton/SegmentedButton.tsx:3]: build passed

**Pull request merged**